nss-day-cohort-43/nuttyshell-picnic-blanket
# Bl events

# Description
This pull request tackles the events issue.  Each user should be able to see only their events.  The events should be displayed in order of their start date and the earliest event should be bigger and have a different background color.  The user should be able to add new events.  They should also be able to delete events.  Events that are within 5 days of the current date should be able to display the max temperature for the date of that event.
Fixes # (issue)
## Type of change
Please delete options that are not relevant.
# Testing Instructions
- Log in as different users and make sure that only their events display
- Add multiple events to different users and make sure they display correctly (also make sure you give valid zip codes, or else the weather fetch will cause an error)
- Try deleting events
- Make sure that only events that are within 5 days will display the weather button
- Test the weather button for each event that has it
# Checklist:
- [x ] My code follows the style guidelines of this project
